doc/semieunit.xml

```diff
@@ -0,0 +1,292 @@
+<digraph with 5 vertices, 11 edges>
+gap> y := InducedSubdigraph(x, [1, 2, 3]);
+<digraph with 3 vertices, 6 edges>
+gap> A := AutomorphismGroup(x);
+Group([ (2,4)(3,5) ])
+gap> S := McAlisterTripleSemigroup(A, OnPoints, x, y);
+<McAlister triple semigroup over Group([ (2,4)(3,5) ])>
+gap> T := McAlisterTripleSemigroup(A, x, y);
+<McAlister triple semigroup over Group([ (2,4)(3,5) ])>
+gap> T = S;
+true
+gap> a := MTE(S, 1, (2, 4)(3, 5));
+(1, (2,4)(3,5))
+gap> a * a;
+(1, ())
+gap> gens := GeneratorsOfSemigroup(S);
+[ (1, ()), (2, ()), (3, ()), (1, (2,4)(3,5)) ]
+gap> P := AsSemigroup(IsPartialPermSemigroup, S);
+<inverse partial perm monoid of size 4, rank 4 with 4 generators>
+gap> IsomorphismMcAlisterTripleSemigroup(P);
+MappingByFunction( <inverse partial perm monoid of size 4, rank 4 
+ with 4 generators>, <McAlister triple semigroup over Group([ (), (), (1,
+2) ])>, function( s ) ... end )
+gap> AsMcAlisterTripleSemigroup(P);
+<McAlister triple semigroup over Group([ (), (), (1,2) ])>
+  ]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="EUnitaryInverseCover">
+  <ManSection>
+    <Prop Name = "EUnitaryInverseCover" Arg = "S"/>
+    <Returns>A homomorphism between semigroups.</Returns>
+    <Description>
+      If the argument <A>S</A> is an inverse semigroup then this function
+      returns a finite E-unitary inverse cover of <A>S</A>. A finite E-unitary
+      cover of <A>S</A> is a surjective idempotent separating homomorphism from
+      a finite E-unitary inverse semigroup to <A>S</A>. A semigroup
+      homomorphism is said to be idempotent separating if no two idempotents
+      are mapped to the same element of in the image.
+<Example><![CDATA[
+gap> S := InverseSemigroup([PartialPermNC([1, 2], [2, 1]),
+> PartialPermNC([1], [1])]);
+<inverse partial perm semigroup of size 7, rank 2 with 2 generators>
+gap> cov := EUnitaryInverseCover(S);
+MappingByFunction( <inverse partial perm semigroup of rank 4 with 2 generators>,
+<inverse partial perm semigroup of size 7, rank 2 with 2 generators>
+ , function( x ) ... end )
+gap> IsEUnitaryInverseSemigroup(Source(cov));
+true
+gap> S = Range(cov);
+true]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="IsFInverseSemigroup">
+  <ManSection>
+    <Prop Name = "IsFInverseSemigroup" Arg = "S"/>
+    <Returns><K>true</K> or <K>false</K>.</Returns>
+    <Description>
+      This function determines whether a given semigroup is an F-inverse
+      semigroup. F-inverse semigroup are a class of E-unitary inverse
+      semigroups. A McAlister triple semigroup
+      <C>McAlisterTripleSemigroup(G,act,X,Y)</C> is F-inverse precisely when
+      the partial order digraph <C>X</C> is a join-semilattice digraph. This
+      function uses <Ref Oper = "AsMcAlisterTripleSemigroup"/> to find a
+      McAlister triple semigroup isomorphic to <A>S</A> then checks if the
+      McAlister triple semigroup is F-inverse.
```

Comment:
Actually this doesn't work. 

`McAlisterTripleSemigroup(Group((4,5)), Digraph([[1],[1,2],[1,2,3],[1,2,3,4],[1,2,3,5]]), [1 .. 4]);`

is not an F-inverse semigroup since the 2nd argument is not a join-semilattice yet the partial order of D-classes

`[ [ 1, 2, 3, 4 ], [ 2 ], [ 2, 3 ], [ 2, 4 ] ]`

defines a join-semilattice digraph.